### DISCUSSION TITLE
Database locally - Database Ciqual

### DISCUSSION THREAD

**Discussion by zuli3n** [wger-project/wger#1752] [Ideas]
Good evening, 
Would it be possible to add a food with its macro-nutrients yourself, locally, I see that there are quite a few errors on OFF. And finally, in terms of ideas, would it be possible to import the ciqual database (https://ciqual.anses.fr/#/cms/telechargement/node/20)
Thanking you for your feedback.

Best regards.

**Comment by rolandgeider** [comment]
Have opened issue https://github.com/wger-project/wger/issues/1758 for this
